packages/react/src/components/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -87,10 +88,11 @@ export interface RadioButtonGroupProps
    * the group changes
    */
   onChange?: (
-    selection: React.ReactNode,
```

Comment:
The `selection` will be the same as the `RadioButton` value. `ReactNode` is too broad.